snmp/tests/test_profiles.py

```diff
@@ -65,16 +65,28 @@ def test_load_profiles(caplog):
         caplog.clear()
 
 
-def run_profile_check(recording_name):
+def run_profile_check(recording_name, profile=None):
     """
     Run a single check with the provided `recording_name` used as
     `community_string` by the docker SNMP endpoint.
     """
+
     instance = common.generate_instance_config([])
 
     instance['community_string'] = recording_name
     instance['enforce_mib_constraints'] = False
     check = SnmpCheck('snmp', {}, [instance])
+
+    # First, see if recording name is a profile, then use profile as definition.
+    prof_def = check.profiles.get(recording_name)
+    if profile:
+        prof_def = check.profiles.get(profile)
+    if prof_def:
```

Comment:
should we log that we’re skipping profile validation?

Reply:
Sure!

Reply:
It turns out a warning will be logged even if the profile isn't specifically validated, which will signify the error:
`WARNING  datadog_checks.base.checks.base.snmp:base.py:716 When specifying a table, you must specify a list of symbols`